stestr/commands/list.py

```diff
@@ -37,22 +37,37 @@ def get_parser(self, prog_name):
                             "filtering specified also uses it)")
         parser.add_argument('--blacklist-file', '-b',
                             default=None, dest='blacklist_file',
-                            help='Path to a blacklist file, this file '
+                            help='DEPRECATED: This option will soon be  '
+                                 'replaced by --exclude-list which is '
+                                 'functionally equivalent.')
+        parser.add_argument('--exclude-list', '-e',
+                            default=None, dest='exclusion_list_file',
+                            help='Path to an exclusion list file, this file '
                                  'contains a separate regex exclude on each '
                                  'newline')
         parser.add_argument('--whitelist-file', '-w',
                             default=None, dest='whitelist_file',
-                            help='Path to a whitelist file, this file '
+                            help='DEPRECATED: This option will soon be  '
+                                 'replaced by --include-list which is '
+                                 'functionally equivalent.')
+        parser.add_argument('--include-list', '-i',
+                            default=None, dest='inclusion_list_file',
+                            help='Path to an inclusion list file, this file '
                                  'contains a separate regex on each newline.')
         parser.add_argument('--black-regex', '-B',
                             default=None, dest='black_regex',
+                            help='DEPRECATED: This option will soon be  '
+                            'replaced by --exclusion-regex which is '
+                            'functionally equivalent.')
+        parser.add_argument('--exclusion-regex', '-E',
```

Comment:
Instead of `--exclusion-regex` I think `--exclude-regex` is probably a better term. "exclusion" makes more sense in English  but exclude is less typing it also matches `--exclude-list`

Reply:
Agreed :smile:
- fixing this across the files changes and
- updating [Issue#296](https://github.com/mtreinish/stestr/issues/296) text also